crime-evidence/src/main/java/uk/gov/justice/laa/crime/evidence/repository/IncomeEvidenceRequiredRepository.java

```diff
@@ -10,21 +10,20 @@ public interface IncomeEvidenceRequiredRepository extends CrudRepository<IncomeE
     @Query(
         value =
             """
-                select ID,
-                       EVIDENCE_ITEMS_REQUIRED
-                from   income_evidence_required i
+                select *
```

Comment:
Is selecting everything here intentional, or should we still only be selecting `id` and `evidence_items_required`?

Reply:
Was throws exception as we return IncomeEvidenceRequiredEntity, hence we changed to *.

Reply:
What exception was it throwing? I remember doing the work to migrate this code from the stored procedure (which for this particular query was just taking the query verbatim and moving it into code), only those two fields were being returned.

Reply:
Thanks for showing me where this was going wrong. I don't understand really why `main` isn't broken (and the tests) given that there haven't been any changes to the entity class or how we're using it (for reference, the query was brought across without modification from the db and the code consuming it hasn't changed either).

In the interests of avoiding the rework now I'm happy to roll with your change of selecting `*` from the table just to get the additional required fields back, this is something to put on the backlog to address later.